docs/3.0/resources/upgrade-to-prefect-3.mdx

```diff
@@ -169,3 +169,66 @@ This change affects you if: You're using agents from an early version of Prefect
 </info>
 
 In Prefect 2, agents were deprecated in favor of next-generation workers. Workers are now standard in Prefect 3. For detailed information on upgrading from agents to workers, please refer to our [upgrade guide](https://docs-3.prefect.io/3.0/resources/upgrade-agents-to-workers).
+
+### Resolving common gotchas
+
+#### `AttributeError: 'coroutine' object has no attribute <some attribute>`
+
+If you do **not** `await` an asynchronous function or method, this error will be raised when you try to use the object.
```

Comment:
I think some context here would be helpful, such as "if you do not await within an async task or flwo context"